File: exemplo.java
public class Main { 
  public static int somar(int a, int b) { 
    return a + b; 
  } 
  public static void main(String[] args) { 
    int total = 0; 
    for (int i = 0; i < 4; i++) {  // Isso vai iterar i de 0 a 3 
      total = somar(total, i); 
    } 
    System.out.println(total); 
  } 
}

File: exemplo.py
def somar(a, b): 
  return a + b 
  
# Exemplo de uso de um laço FOR com range 
total = 0 
for i in range(4): # Isso vai iterar i de 0 a 3 
  total = somar(total, i) 
  
print(total)
